include/seqan3/alignment/aligned_sequence/debug_stream_alignment.hpp

```diff
@@ -94,10 +96,8 @@ namespace seqan3
  * \return The given stream to which the alignment representation is appended.
  */
 template <typename char_t, typename alignment_t>
-//!\cond
-    requires (detail::debug_streamable_tuple<alignment_t> &&
-              detail::all_model_aligned_seq<detail::tuple_type_list_t<std::remove_cvref_t<alignment_t>>>)
-//!\endcond
+    requires(detail::debug_streamable_tuple<alignment_t> &&
```

Comment:
can we keep the space after the requires?

Reply:
works with clang-format 15